### DISCUSSION TITLE
discuss: allow applications after application end date in a round

### DISCUSSION THREAD

**Discussion by thelostone-mc** [gitcoinco/grants-stack#421] [Q&A]
#### Description

The question here is currently within in round contract, projects can apply even after the round has ended / after the application date. This was originally done as opposed to having safe guard logic at the contract level.

The discussion here is to understand the need to enforce these restrictions at the round contract level and to ensure it does not inhibit to support a new kind of round in the future where applications should be allowed during the round as well 

https://discord.com/channels/562828676480237578/1021916781520097350/1022433111058960406

**Comment by michellema1208** [comment]
posting my comment from Discord: <@583676649468592129> since when you’re setting up a round, you can specify an application end date and a round start date separately, could Round Manager just validate against the specified application end date?

So if a round operator wanted to accept applications after the round start date, when they set up a round, they could just specify that their application end date is after the round start date

**Reply by thelostone-mc** [reply]
That's right. We could enforce this kind of behavior either at 

- contract
- subgraph 
- frontend 

Like you mentioned -> keeping it at the frontend ensure we can support rounds where folks wanted to create a round where applications are  allowed at any point of time (though i don't know if it makes sense)

**Reply by vacekj** [reply]
Have we reached consensus on this outside of this discussion?
